react/react1/week2/todo-list/src/List.jsx

```diff
@@ -0,0 +1,31 @@
+import React,{ useState}  from 'react';
+
+export default function TodoList  ({ list, onDelete }) {
+    return(
+        <ul>
+        {list.map((task) => (
+          <Task key={task.id} task={task} onDelete={onDelete} />
+        ))}
+      </ul>
+      );    
+} 
+    
+      function Task ({ task, onDelete })  { 
+        //const [checked,setChecked]=useState(false)
+        //style={{textDecoration: 'line-through'}}
```

Comment:
you can pass this state as props to the CSS and style when the check is true
hint: you'll need a ternary operator to do that.